README.md

```diff
@@ -93,6 +93,27 @@ c93da9ff...	172.17.8.103    role=kubernetes
 if you are using Parallels Desktop and the [vagrant-parallels](http://parallels.github.io/vagrant-parallels/docs/) provider
 just add ```--provider parallels``` to the ```vagrant up``` invocations above
 
+## Costumization
+
+all aspects of your cluster setup can be costumized with environment variables. right now the available ones are:
+
+ - **NUM_INSTANCES** will set the number of nodes (minions).
+   if unset defaults to 2
+ - **UPDATE_CHANNEL** will set the default CoreOS channel to be used in the VMs. the default is the **alpha** channel (alternatives are **stable** and **beta**).
+ - **COREOS_VERSION** will set the specific CoreOS release (from the set channel) to be used. The default is to use whatever is the latest one from the given channel.
+ - **SERIAL_LOGGING** if set to true will allow logging from the VMs serial console. defaults to false. Only allow it if you *really* know what you are doing.
+ - **MASTER_MEM** sets the master's VM memory. Defaults to 512 (MB)
+ - **MASTER_CPUS** sets the number os vCPUs to be used by the master's VM. Defaults to 1.
+ - **NODE_MEM** sets the minions's VM memory. Defaults to 1024 (MB)
+ - **NODE_CPUS** sets the number os vCPUs to be used by the minions's VMs. Defaults to 1.
+ - **KUBERNETES_VERSION** defines the specific kubernetes version being used. *If* the [world](http://google.com/about) was perfect we'd be using by default the latest and greatest one, as it [isn't](https://github.com/GoogleCloudPlatform/kubernetes/issues/4415) currently we are defaulting to 0.9.3. 
+
+so, in order to start, say, a cluster based on CoreOS's stable channel one just would do...
```

Comment:
Can you do proper capitalization? I'm nitpicking here but I take it seriously when writing documentation.